Firestore/Example/Tests/Util/FSTIntegrationTestCase.mm

```diff
@@ -14,10 +14,9 @@
  * limitations under the License.
  */
 
-@import Firestore;
```

Comment:
Why did this change? Isn't the Firestore module still available? Presumably related to the Objective C++ conversion?

Reply:
Objective-C modules are not available from Objective-C++ without the additional `-fcxx-modules` flag.

I initially steered away from this because the [clang 3.5.1 docs](http://releases.llvm.org/3.5.1/tools/clang/docs/Modules.html) call `-fcxx-modules` "EXPERIMENTAL and VERY BROKEN".

The [clang 3.7.0 docs](http://releases.llvm.org/3.7.0/tools/clang/docs/Modules.html) (roughly [corresponding to Xcode 7.0](https://gist.github.com/yamaya/2924292)) no longer call this out.

However, based on [discussion on the clang lists](http://lists.llvm.org/pipermail/cfe-commits/Week-of-Mon-20150309/125146.html) it seems like the mention of `-fcxx-modules` and its experimental nature was removed because it was scary, not because it actually got much better.

Passing this flag works in Xcode 8.3.3 but given how far back our support horizon goes I'm disinclined toward using the feature.

Reply:
Makes sense.